demos/security-trifecta/src/pages/login/UnlockCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { IonButton, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle } from '@ionic/react';
+import { useHistory } from 'react-router-dom';
+import { login } from '../../utils/auth';
+
+const UnlockCard: React.FC = () => {
+  const history = useHistory();
+
+  const unlock = async () => {
+    try {
+      await login();
```

Comment:
OK, that explains the before mentioned "unlock appears to do a login" comment.

Reply:
Resolved. I think I know where copying my changes over went wrong now